tests/v2/validation/provisioning/rke2/agent_customization_test.go

```diff
@@ -43,6 +44,9 @@ func (r *RKE2AgentCustomizationTestSuite) SetupSuite() {
 
 	r.client = client
 
+	r.provisioningConfig.RKE2KubernetesVersions, err = kubernetesversions.Default(r.client, clusters.RKE2ClusterType.String(), r.provisioningConfig.RKE2KubernetesVersions)
```

Comment:
Why is this one being left alone to have the default version?

Reply:
agentCustomization shouldn't be impacted by k8s version minor bumps. It seemed like overkill to run it on all available k8s versions but if you want it done that way I can ruit on all available rke2 types.

Reply:
No need, thanks for the clarification.